mllib/src/test/scala/org/apache/spark/ml/source/libsvm/LibSVMRelationSuite.scala

```diff
@@ -88,7 +88,8 @@ class LibSVMRelationSuite extends SparkFunSuite with MLlibTestSparkContext {
     val df = sqlContext.read.format("libsvm").load(path)
     val tempDir2 = Utils.createTempDir()
     val writepath = tempDir2.toURI.toString
-    df.write.format("libsvm").mode(SaveMode.Overwrite).save(writepath)
+    // TODO: Remove requirement to coalesce by supporting mutiple reads.
+    df.coalesce(1).write.format("libsvm").mode(SaveMode.Overwrite).save(writepath)
```

Comment:
Same here. What does "multiple reads" mean here and why `coalesce(1)` is required? `LibSVMRelation` uses `textFiles` to read LibSVM files under the hood, so I'd assume that it can read from multiple part-files?